File: UC-3/Exercícios_basicos_Python/Exercicio_4.py
# Crie um programa que receba três valores do usuário.
# Imprima a soma dos dois primeiros multiplicada pelo terceiro.
num1 = int(input("Digite o primeiro número: "))
num2 = int(input("Digite o segundo número: "))
num3 = int(input("Digite o terceiro número: "))

print((num1+num2)*num3)

# Faça um programa em linguagem Python que converta
# metros para centímetros. 
centimetros = float(input("Digite a metragem em cm: "))
metros = centimetros / 100
print(metros)

# Faça um algoritmo em linguagem Python que o nome,
# idade, sexo, e receba duas notas e calcule a média
# aritmética e mostre o resultado.
nome = str(input("Digite seu nome: "))
idade = int(input("Digite sua idade: "))
sexo = str(input("Digite seu sexo: "))
nota1 = int(input("Digite a primeira nota: "))
nota2 = int(input("Digite a segunda nota: "))
media = (nota1 + nota2) / 2
print(f"Nome: {nome}, Idade: {idade}, Sexo: {sexo}, Média: {media}")

File: UC-8_P.O.O/Introducao_Java/src/Exercicio4.java
import java.util.Scanner;

public class Exercicio4 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite o primeiro número: ");
        double num1 = scanner.nextDouble();
        
        System.out.print("Digite o segundo número: ");
        double num2 = scanner.nextDouble();
        
        System.out.print("Digite o terceiro número: ");
        double num3 = scanner.nextDouble();
        
        if (num1 < num2 && num2 < num3) {
            System.out.println("Os números estão em ordem crescente");
        } else {
            System.out.println("Os números NÃO estão em ordem crescente");
        }
        
        scanner.close();
    }
}